2.9/index.xml

```diff
@@ -1008,106 +1008,6 @@ Task AbortTransactionAsync(CancellationToken cancellationToken = default(Cancell
 </description>
     </item>
     
-    <item>
```

Comment:
As previously noted, for some reason the xml files were generated with their contents in a slightly different order.

Reply:
Hugo...